src/backend/assets/scenarios/scenario1.json

```diff
@@ -0,0 +1,20 @@
+{
+"tasks": [
+        {
+            "action": "open",
+            "target": "zawur1 mordo",
+            "value": 100,
```

Comment:
co oznacza value? I czy target to referencja po nazwie komponentu? Czy jego ID? Potrzebny byłby w takim razie "target type" lub coś tego typu, prawda?

Reply:
at the end chce żeby nazwa i workstacja była klucze głównym, więc nazwa jest unique dla danej stacji, więc target to bedzie nazwa, a open bedzie czymś w stylu set_open 0-closed, 1 opened